core/src/test/java/nu/yona/server/test/util/JUnitUtil.java

```diff
@@ -46,7 +47,7 @@ private JUnitUtil()
 	public static <T, ID extends Serializable> void setUpRepositoryMock(CrudRepository<T, ID> mockRepository)
 	{
 		// save should not return null but the saved entity
-		when(mockRepository.save(Matchers.<T> any())).thenAnswer(new Answer<T>() {
+		Mockito.lenient().when(mockRepository.save(ArgumentMatchers.<T> any())).thenAnswer(new Answer<T>() {
```

Comment:
Why is `lenient` required here?

Reply:
We are adding a couple of repositories and on some, ``save`` is being called, but on others not. Without ``lenient``, Mockito complains about these "others".